snap/snapcraft.yaml

```diff
@@ -26,34 +33,34 @@ apps:
       - x11
       - gsettings
 
-parts:
+plugs:
+  gtk-3-themes:
+    interface: content
+    target: $SNAP/share/themes
+    default-provider: gtk-common-themes:gtk-3-themes
+  icon-themes:
+    interface: content
+    target: $SNAP/share/icons
+    default-provider: gtk-common-themes:icon-themes
+  kde-frameworks-5-plug:
+    interface: content
+    target: kf5
+    default-provider: kde-frameworks-5-core18:kde-frameworks-5-core18-slot
```

Comment:
`icon-themes` and `kde-frameworks-5-plug` are also not needed anymore since the `kde-neon` extension adds them.

Reply:
Note: you can run `snapcraft expand-extensions` to see what the resulting snapcraft.yaml will be after the extensions do their thing. The resulting snapcraft.yaml should be identical whether these are added or not.